src/sdk/pynni/nni/hyperopt_tuner/hyperopt_tuner.py

```diff
@@ -51,6 +51,8 @@ def json2space(in_x, name=NodeType.ROOT):
             _value = json2space(in_x[NodeType.VALUE], name=name)
             if _type == 'choice':
                 out_y = eval('hp.hp.choice')(name, _value)
+            elif _type == 'randint':
+                out_y = hp.hp.uniform(name, *_value)
```

Comment:
this is not correct.

Reply:
Why? This is a workaround since `randint` in hyperopt has strange behaviors (returning float for example) and do not have lower bound.